src/main/java/com/google/sps/data/Activity.java

```diff
@@ -0,0 +1,21 @@
+package com.google.sps.data;
+
+public class Activity {
+  private String name;
+  private String type;
```

Comment:
What is type? Should this be an enum or int instead of a String?

Reply:
To define the types were calling them [instead name]_CATEGORY. Should we call this category instead?

Reply:
Fixed it.